component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1828,10 +1835,9 @@ private void publishPostSMSOTPGeneratedEvent(HttpServletRequest request, Authent
             eventProperties.put(IdentityEventConstants.EventProperty.OTP_GENERATED_TIME, otpGeneratedTime);
 
             String otpExpiryDuration = SMSOTPUtils.getTokenExpiryTime(context);
```

Comment:
Here we have used the term "duration". AFAIU, this is the 900 seconds we set. Let's stick to one simple wording.

IMO, duration sounds better.

Reply:
Fixed